### ISSUE TITLE
Otter Grader downloads chromium on gradescope

### ISSUE THREAD

**Issue by TylerADavis** [ucbds-infra/otter-grader#38]
Though [execute_notebook contains logic to mock out PDF generation](https://github.com/ucbds-infra/otter-grader/blob/1e057b1630ce0aa191689017a7a557aaf045682b/otter/grade.py#L240), I'm still seeing logs indicating that Chromium is being downloaded. While this doesn't break anything, it clutters the logs and probably adds some time to autograder execution.

Perhaps the import chain of otter -> nb2pdf -> nbpdfexport -> ? somehow automatically starts the download

My notebooks look like the following
```python
import otter
grader = otter.Notebook()
grader.export("proj3.ipynb", filtering=False)
```

Gradescope logs, note that the output relating to chromium comes before the student output
```
[W:pyppeteer.chromium_downloader] start chromium download.
Download may take a few minutes.
[W:pyppeteer.chromium_downloader] 
chromium download done.
[W:pyppeteer.chromium_downloader] chromium extracted to: /root/.local/share/pyppeteer/local-chromium/575458
We filtered out 92% of the entries

Duration: 240.3395219270353 
Speed: 226.90793945018308



  name  score  possible visibility
0  q1b    2.0         2    visible
1  q1c    3.0         3    visible
2  q2a    1.0         1    visible
```

**Comment by chrispyles** [comment]
There was a missing patch of `otter.Notebook.export` that might have been causing this. It is fixed in #49; will see if this keeps happening in v0.4.5

**Comment by chrispyles** [comment]
Based on the latest test this appears to have fixed it

**Comment by chrispyles** [comment]
Fixed by #49. Will release v0.4.5 tonight and this fix will be made.
